### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getGrammar' of undefined

### ISSUE THREAD

**Issue by adamvi** [felixjung/repl-term#3]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.2
**System**: Mac OS X 10.10.4
**Thrown From**: [repl-term](https://github.com/felixjung/repl-term) package, v0.1.3

### Stack Trace

Uncaught TypeError: Cannot read property 'getGrammar' of undefined

```
At /Users/avi/.atom/packages/repl-term/lib/language-handler.coffee:6

TypeError: Cannot read property 'getGrammar' of undefined
    at Object.module.exports.currentLanguage (/Users/avi/.atom/packages/repl-term/lib/language-handler.coffee:8:54)
    at Object.module.exports.launchRepl (/Users/avi/.atom/packages/repl-term/lib/terminal-handler.coffee:47:30)
    at atom-workspace.subscriptions.add.atom.commands.add.repl-term:launch-repl (/Users/avi/.atom/packages/repl-term/lib/repl-term.coffee:13:34)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:156:19)
    at EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:70:30)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:166:7)
```

### Commands

```
     -0:00.7.0 repl-term:launch-repl (div.settings-view.pane-item)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
language-r, v0.2.0
repl-term, v0.1.3
run-in-terminal, v0.3.0

# Dev
No dev packages
```

**Comment by felixjung** [comment]
Right, I reproduced this just now. You need to have at least one text editor window (by that I mean a text buffer) open to use the package. You can create any new file or open an existing one and then try again. I need to catch this and give some GUI feedback to the user.

Thanks for reporting!

**Comment by felixjung** [comment]
This has been fixed by 01f244cb9cd6fa3003243f6b89ab9de3a97ed6e1. When using any of the commands to launch or send to a REPL with no active editor, the user is presented with an according notification to first open a file.
